pyproject.toml

```diff
@@ -106,12 +104,8 @@ http2 = [
     "priority >= 1.1.0, < 2.0",
 ]
 
-contextvars = [
```

Comment:
maybe we should still keep this extra dependency ... but leave it empty for compatibility.

Maybe we can add a note about this in the release notes for 2023.8.0

-----------

If we remove this, if someone tries to `pip install twisted[contextvars]`  with this change  it will fail..

But maybe is ok to have these type of breaking changes... I don't know how to best handle this.

Reply:
It's not a breaking change, if you attempt to install an extra that doesn't exist you just get a warning, and the install completes successfully:
```bash
$ pip install twisted[banana]
Collecting twisted[banana]
  Using cached Twisted-22.10.0-py3-none-any.whl (3.1 MB)
WARNING: twisted 22.10.0 does not provide the extra 'banana'
```

Reply:
Thanks for the info. all good.